sklearn/utils/estimator_checks.py

```diff
@@ -1327,7 +1439,7 @@ def fit(self, X, y):
     if name in ('FeatureUnion', 'Pipeline'):
         e = estimator([('clf', T())])
 
-    elif name in ('GridSearchCV' 'RandomizedSearchCV'):
```

Comment:
Yes I guess neither 'GridSearchCV' nor 'RandomizedSearchCV' did in fact trigger an error on this test